<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>signup</class>
 <widget class="QMainWindow" name="signup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sign Up</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
    QMainWindow {
        background-color: qlineargradient(x1:0, y1:0, x2:1, y2:1,
                                          stop:0 #f0f4f8, stop:1 #dbeafe);
    }
   </string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>36</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="minimumSize">
           <size>
            <width>291</width>
            <height>111</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">
            font: bold 36pt &quot;Segoe UI&quot;;
            color: #1e3a8a;
            padding: 20px;
            background-color: #ffffff;
            border-radius: 15px;
           </string>
          </property>
          <property name="text">
           <string>Sign Up</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>13</width>
          <height>132</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="styleSheet">
             <string notr="true">font: 15pt &quot;Segoe UI&quot;;
color: black;</string>
            </property>
            <property name="text">
             <string>Username</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="username_input">
            <property name="minimumSize">
             <size>
              <width>670</width>
              <height>35</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 12pt &quot;Segoe UI&quot;;
color: black;
background-color: #f0f0f0;
border: 1px solid #ccc;
border-radius: 6px;
padding: 5px 8px;</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="styleSheet">
             <string notr="true">font: 15pt &quot;Segoe UI&quot;;
color: black;</string>
            </property>
            <property name="text">
             <string>Password</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="password_input">
            <property name="minimumSize">
             <size>
              <width>638</width>
              <height>35</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 12pt &quot;Segoe UI&quot;;
color: black;
background-color: #f0f0f0;
border: 1px solid #ccc;
border-radius: 6px;
padding: 5px 8px;</string>
            </property>
            <property name="echoMode">
             <enum>QLineEdit::EchoMode::Password</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Eye_Button">
            <property name="minimumSize">
             <size>
              <width>35</width>
              <height>35</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
    background-color: transparent;
    border: none;
    border-image: url(:/images/images/closed_eye.png);
}
QPushButton:checked {
    border-image: url(:/images/images/open_eye.png);
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="0" colspan="2">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QPushButton" name="Back_Button">
            <property name="minimumSize">
             <size>
              <width>251</width>
              <height>52</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">
              QPushButton {
                  background-color: #1d4ed8;
                  color: white;
                  font: 14pt &quot;Segoe UI&quot;;
                  border-radius: 10px;
                  padding: 10px;
              }
              QPushButton:hover {
                  background-color: #2563eb;
              }
              QPushButton:pressed {
                  background-color: #1e40af;
              }
             </string>
            </property>
            <property name="text">
             <string>Back</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="LogIn_Button">
            <property name="minimumSize">
             <size>
              <width>251</width>
              <height>52</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">
              QPushButton {
                  background-color: #1d4ed8;
                  color: white;
                  font: 14pt &quot;Segoe UI&quot;;
                  border-radius: 10px;
                  padding: 10px;
              }
              QPushButton:hover {
                  background-color: #2563eb;
              }
              QPushButton:pressed {
                  background-color: #1e40af;
              }
             </string>
            </property>
            <property name="text">
             <string>Sign Up</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>13</width>
        <height>131</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
